### PR TITLE
added one english quote

### PR DESCRIPTION
**Pull request by Jitesh117**
Added one english quote from Halo3

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -35640,6 +35640,12 @@
       "source": "Steppenwolf",
       "length": 52,
       "id": 6094
+    },
+    {
+      "text": "They let me pick. Did I ever tell you that? Choose which ever Spartan I wanted. You know me. I did my research, watched as you became the soldier we needed you to be. But you had something they didn't, something no one saw but me. Can you guess? Luck.",
+      "source": "Halo3",
```
[the remark this answers is not in the document]
Comment (Jitesh117) [reply frontend/static/quotes/english.json:35646]: True. I have changed that now

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -35640,6 +35640,12 @@
       "source": "Steppenwolf",
       "length": 52,
       "id": 6094
+    },
+    {
+      "text": "They let me pick. Did I ever tell you that? Choose which ever Spartan I wanted. You know me. I did my research, watched as you became the soldier we needed you to be. But you had something they didn't, something no one saw but me. Can you guess? Luck.",
```
Comment (Bruception) [inline frontend/static/quotes/english.json:35645]: This quote seems incomplete:
https://www.imdb.com/title/tt0494232/characters/nm1071768

I'm not sure what @Miodec's policy is on this

Comment (Jitesh117) [reply frontend/static/quotes/english.json:35645]: Now the quote is complete. Sorry for pushing the incomplete quote

**Review by Bruception — APPROVED** [review]
Lgtm

**Pull request merged**